spec/controllers/api/v2/app_identifier_visitor_configs_controller_spec.rb

```diff
@@ -0,0 +1,85 @@
+require 'rails_helper'
+
+RSpec.describe Api::V2::AppIdentifierVisitorConfigsController do
+  describe "#show" do
+    let(:app) { FactoryBot.create(:app) }
+    let(:identifier_type) { FactoryBot.create(:identifier_type, name: "clown_id") }
+    let(:feature_gate) { FactoryBot.create(:feature_gate, name: "blab_enabled", registry: { false: 50, true: 50 }) }
+    let(:visitor) { FactoryBot.create(:visitor) }
+    let(:identifier) { FactoryBot.create(:identifier, visitor: visitor) }
+
+    it "has knocked-out weightings and it doesn't include a non-force assignment for a feature gate that isn't feature complete" do
+      FactoryBot.create(
+        :assignment,
+        visitor: visitor,
+        split: feature_gate,
+        variant: "true",
+        context: "the_context",
+        mixpanel_result: "success"
```

Comment:
we can probably elide context and mixpanel result in factory config now that they're not returned?